docs/collections/index.rst

```diff
@@ -8,16 +8,16 @@ The following contain the regolith schemas and examples in both YAML and JSON/Mo
 .. toctree::
     :maxdepth: 1
 
-    abstracts
-    assignments
-    blog
+    auto/abstracts
+    auto/assignments
+    auto/blog
     citations
     courses
-    grades
-    grants
+    auto/grades
+    auto/grants
     jobs
     news
```

Comment:
Why are these not generated?

Reply:
I haven't made schemas for them yet

Reply:
haha OK - should that be a requirement for this PR? Or do you want to delay it?

Reply:
I was hoping to delay it. I'm expecting a couple more takes at this as I start using more of the dbs so I might come around to it.

Reply:
Sounds good to me!